### PR TITLE
Add Topic Modelling video.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: content/pages/programme.rst [hunk 1]
```diff
@@ -714,7 +714,9 @@ Vous trouverez ci-dessous le programme de la Pycon-Fr 2016.
...
           Gensim is an open-source python NLP framework which provides an API to do robust, industry-grade Topic Modelling which is memory independent and super fast, while being very simple to use.
           The best part of gensim and python for Topic Modelling is it’s ease of usage and effectiveness. I would propose a small talk to explain how to effectively do topic modelling in python using Gensim framework- especially - after identifying topics from a large dataset, and then leveraging to perform un-supervised clustering, colouring topic-words in a document, and better understanding textual data for subsequent usage. All of this will be supported with examples from research and industry.
-           [ My relationship with Gensim is through the Google Summer of Code 2016 program, where I am implementing Dynamic Topic Models for them ]</div>
+           [ My relationship with Gensim is through the Google Summer of Code 2016 program, where I am implementing Dynamic Topic Models for them ]
+		   <br /><a href="http://www.demo.openveo.com/publish/video/r19ZYySke?fullscreen&lang=en">Watch the video</a>
+		   </div>
```
Comment (almet) [inline content/pages/programme.rst:719]: If possible I believe we should host the video files / or at least the html ourselves.what happens if this demo server goes down?

Comment (Natim) [reply content/pages/programme.rst:719]: The video are hosted on Youtube, we can host the player somewhere else but lets iterate, my goal is to add as many video as possible and then we will be able to change the URL on the website if needed.

**Comment by ygravrand** [comment]
I strongly believe that we should make all videos available at the same time, otherwise it is somewhat unfair.
Some people asked me to be notified, too. We could use the list of people who signed up.

**Comment by Natim** [comment]
> otherwise it is somewhat unfair.

It will be unfair anyway because we don't have all of them, many will take more time to process than others. I don't think people will watch all the video so we should put the one we have asap.

File: content/pages/programme.rst (hunk 1 above)
Comment (ygravrand) [reply content/pages/programme.rst:719]: Plus, the original video on youtube doesn't belong to us and isn't properly tagged / referenced.

Comment (ygravrand) [reply content/pages/programme.rst:719]: That said, the system is great.
The best way would be to integrate openveo player in the site, it supports simple video tags... But it's a lot of work

**Comment by ygravrand** [comment]
When do we notify people, then?

**Comment by Natim** [comment]
We could do a tweet per talk and a global notification at the end.

**Comment by ygravrand** [comment]
Fine, I'll go with that.
There is still a problem with the youtube channel / referencing / ownership etc.
Are there any options in the openveo uploader?
I would prefer uploading videos ourselves so that we keep ownership (including taking them down, ...) and reference them from the tool.

**Comment by ygravrand** [comment]
The "self hosting + openveo on the site" is only the icing on the cake ;-)

**Comment by Natim** [comment]
Ok it seems that we need to install the tool ourselves then.

**Comment by ygravrand** [comment]
Can't we just do a regular youtube upload and provide the xml + jpgs separately?

**Comment by ygravrand** [comment]
Hmm, another problem is that the videos won't include the slides anyway. So we should link to openveo at least in the youtube description...

**Comment by Natim** [comment]
I will see if we can have a standalone player served in https://2016.pycon.fr/video/video-ID that can load the slides from the same origin.

**Comment by almet** [comment]
I wouldnt go this way. Maybe we should agree on a way to do things first and then do it...

**Comment by ygravrand** [comment]
Agreed: little details but they matter :-)

**Comment by grenoya** [comment]
Please keep in mind that twiter is not used by everyone. At some point, 
a global email should be sent to all speakers (or to all attendant of 
the conference if possible)

**Comment by almet** [comment]
We will of course do that

**Pull request closed without merging**